package.json

```diff
@@ -415,7 +416,8 @@
           "orders-storage.settings.item.get",
           "ui-orders.third-party-services",
           "orders.acquisition-methods.collection.get",
-          "ui-orders.settings.custom-fields.view"
+          "ui-orders.settings.custom-fields.view",
+          "orders-storage.settings.collection.get"
```

Comment:
duplicate https://github.com/folio-org/ui-orders/pull/1597/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R415

Reply:
Correct. I should be happened after the rebase the master